shared/src/domain-logic/opportunityFields.ts

```diff
@@ -1,5 +1,6 @@
 import { LookupLegalStructureById } from "../legalStructure";
 import { ProfileData } from "../profileData";
+import { determineForeignBusinessType } from "./determineForeignBusinessType";
```

Comment:
[sand] Change the import so it isn't a relative import.

Reply:
punting - https://www.pivotaltracker.com/story/show/186365752